test/utils.py

```diff
@@ -637,3 +662,24 @@ def sample_format(self) -> str:
         },
     },
 )
+
+
+# This is a BT.709 full range video, generated with:
+# ffmpeg -f lavfi -i testsrc2=duration=1:size=1920x720:rate=30 \
+# -c:v libx264 -pix_fmt yuv420p -color_primaries bt709 -color_trc bt709 \
+# -colorspace bt709 -color_range pc bt709_full_range.mp4
+#
```

Comment:
Up until this PR, we've maintained the rule that all generated references can be generated from the [generate_reference_resources.sh](https://github.com/pytorch/torchcodec/blob/ffcb7ab2e98c204dfb103f46b2db154cbf1aa713/test/generate_reference_resources.sh) script. Is that something we want to continue to maintain? I think there is a lot of value in it, but that script is also not the cleanest artifact.

Reply:
From what I can tell this script generates the bmp / pt reference *frames*, but not the source video themselves. I see similar comments there indicating how the videos were generated:

https://github.com/pytorch/torchcodec/blob/ffcb7ab2e98c204dfb103f46b2db154cbf1aa713/test/generate_reference_resources.sh#L65-L66